static/src/stylesheets/module/site-messages/_first-pv-consent.scss

```diff
@@ -194,3 +194,19 @@ $btn-height: ($gs-baseline*3.5);
         }
     }
 }
+
+// commercialIabBottomConsentBanner A/B test styles
+
+.cmp-options-button {
+    height: 42px;
+    border: 0;
+    border-radius: 1000px;
+    padding: 0 20px;
+    margin-left: 8px;
+    background: #767676;
+    color: #ffffff;
+
+    :focus {
+        outline: none;
```

Comment:
outline is pretty useful for accessibility reasons.
Is there another visible cue if someone is interacting with the widget via tab navigation?

https://webaim.org/techniques/keyboard/

Reply:
Agree. Will remove.